From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Danila Tikhonov <danila@jiaxyga.com>
Date: Sat, 27 Jul 2024 00:26:07 +0300
Subject: [PATCH] FROMLIST: dt-bindings: pinctrl: qcom,sm8450-tlmm: Add
 compatible for SM8475

Add a dt-bindings compatible string for the Qualcomm Snapdragon 8+
Gen 1 (SM8475) SoC. The SM8475 SoC uses the same driver structure and
functionality as the SM8450 (Snapdragon 8 Gen 1). It also shares
identical properties and pins function names with the SM8450.

Signed-off-by: Danila Tikhonov <danila@jiaxyga.com>
(cherry picked from commit 8be0118129a1f0e43666a81f9a9008ba6c9e66a7)
---
 .../devicetree/bindings/pinctrl/qcom,sm8450-tlmm.yaml     | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/pinctrl/qcom,sm8450-tlmm.yaml b/Documentation/devicetree/bindings/pinctrl/qcom,sm8450-tlmm.yaml
index 16fd2c5e23..8f9536ecda 100644
--- a/Documentation/devicetree/bindings/pinctrl/qcom,sm8450-tlmm.yaml
+++ b/Documentation/devicetree/bindings/pinctrl/qcom,sm8450-tlmm.yaml
@@ -4,20 +4,22 @@
 $id: http://devicetree.org/schemas/pinctrl/qcom,sm8450-tlmm.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: Qualcomm Technologies, Inc. SM8450 TLMM block
+title: Qualcomm Technologies, Inc. SM8450/SM8475 TLMM block
 
 maintainers:
   - Vinod Koul <vkoul@kernel.org>
 
 description:
-  Top Level Mode Multiplexer pin controller in Qualcomm SM8450 SoC.
+  Top Level Mode Multiplexer pin controller in Qualcomm SM8450/SM8475 SoCs.
 
 allOf:
   - $ref: /schemas/pinctrl/qcom,tlmm-common.yaml#
 
 properties:
   compatible:
-    const: qcom,sm8450-tlmm
+    enum:
+      - qcom,sm8450-tlmm
+      - qcom,sm8475-tlmm
 
   reg:
     maxItems: 1
